### PR TITLE
munts_libsimpleio release 1.0.0

### PR DESCRIPTION
**Pull request by pmunts**

### CODE REVIEW

File: index/mu/munts_libsimpleio/munts_libsimpleio-1.0.0.toml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+name = "munts_libsimpleio"
+description = "Linux Simple I/O Library bindings for GNAT Ada"
+version = "1.0.0"
+licenses = "BSD-1-Clause"
+authors = ["Philip Munts"]
+maintainers = ["Philip Munts <phil@munts.net>"]
+maintainers-logins = ["pmunts"]
+website = "http://https://github.com/pmunts/libsimpleio"
```
Comment (mosteo) [inline index/mu/munts_libsimpleio/munts_libsimpleio-1.0.0.toml:8]: Extra http:// here

**Review by mosteo — CHANGES_REQUESTED** [review]
Thanks for your contribution, Philip. I found a typo in the website.

File: index/mu/munts_libsimpleio/munts_libsimpleio-1.0.0.toml (hunk 1 above)
Comment (pmunts) [reply index/mu/munts_libsimpleio/munts_libsimpleio-1.0.0.toml:8]: OK, thanks.  I also realized I need to make this crate available only for Linux.  Can I just edit `index/mu/munts_libsimpleio/munts_libsimpleio-1.0.0.toml` or do I need to republish from the beginning?

Comment (mosteo) [reply index/mu/munts_libsimpleio/munts_libsimpleio-1.0.0.toml:8]: You can push any necessary updates, sure.

**Pull request closed without merging**